source/horoscope/css/landing.css

```diff
@@ -0,0 +1,124 @@
+#landing-title {
+  font-size: 4em;
+  font-weight: bolder;
+  color: white;
+  margin-top: 40px;
+  /* adjust the spacing between the letters */
+  letter-spacing: 0.25em;
+
+}
+
+img {
```

Comment:
Can we not set the background image using css and `background: url(./images.png)` and what ever necessary?

Reply:
that would probably work better imo but I just copied how the horoscope page was done, I can go back and update so it's in the same style as your page

Reply:
makes sense